bindings_generator/src/main.rs

```diff
@@ -31,3 +32,21 @@ fn main() {
         .write_to_file(out_file)
         .expect("Couldn't write bindings!");
 }
+
+use bindgen::callbacks::ParseCallbacks;
+
+#[derive(Debug)]
+struct RenameCallback;
+impl ParseCallbacks for RenameCallback {
+    fn item_name(&self, item: &str) -> Option<String> {
+        match item {
+            v if item.contains("_bindgen_ty_1") => {
+                Some(v.replace("__bindgen_ty_1", "_u").to_string())
+            }
+            v if item.contains("_bindgen_ty_") => {
+                Some(v.replace("__bindgen_ty_", "_u_").to_string())
+            }
```

Comment:
Probably unnecessary, looks like there's no `_ty_2` etc. Maybe this code is cleaner with:

```rust
if let Some(name) = item.strip_suffix("_bindgen_ty_1") {
    Some(format!("{name}_u"))
} else {
    None
}
```

Reply:
(and a `.map`)